cypress/e2e/po/edit/cloud-credentials.po.ts

```diff
@@ -0,0 +1,56 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+import CruResourcePo from '@/cypress/e2e/po/components/cru-resource.po';
+import LabeledSelectPo from '@/cypress/e2e/po/components/labeled-select.po';
+
+export default class CloudCredentialsCreateEditPo extends PagePo {
+  private static createPath(clusterId: string, id?: string ) {
+    const root = `/c/${ clusterId }/manager/cloudCredential`;
+
+    return id ? `${ root }/${ id }` : `${ root }/create`;
+  }
+
+  static goTo(path: string): Cypress.Chainable<Cypress.AUTWindow> {
+    throw new Error('invalid');
+  }
+
+  constructor(clusterId = 'local', id?: string) {
+    super(CloudCredentialsCreateEditPo.createPath(clusterId, id));
+  }
+
+  cloudServiceOptions(): CruResourcePo {
+    return new CruResourcePo('[data-testid="cru-form"]');
+  }
+
+  name(): LabeledInputPo {
```

Comment:
name and description (and in other places namespace) all come from a `NameNsDescription` component, so should be accessed via the `NameNsDescription` PO